hadoop-common-project/hadoop-common/src/main/resources/core-default.xml

```diff
@@ -1457,6 +1457,14 @@
   </description>
 </property>
 
+<property>
```

Comment:
lets actually cut this. Adding default values here increases the size of the configuration when marshalling, which happens a lot these days.

Reply:
done